ConsentViewController/Classes/SourcePointClient/SPNativeUI.swift

```diff
@@ -164,12 +164,10 @@ class SPNativeButton: SPNativeUI {
 class SPNativeImage: SPNativeUI {
     class Settings: SPNativeUISettings {
         let src: String
-        let url: URL?
         
         required init(from decoder: Decoder) throws {
             let container = try decoder.container(keyedBy: Keys.self)
-            src = try container.decode(String.self, forKey: .src)
-            url = try container.decodeIfPresent(URL.self, forKey: .url)
+            src = try container.decodeIfPresent(String.self, forKey: .url) ?? (try container.decode(String.self, forKey: .src))
```

Comment:
Shall we change `src` to be of type `URL?` ?

Reply:
I think we can